rapids-cmake/cpm/nvcomp.cmake

```diff
@@ -95,7 +95,7 @@ function(rapids_cpm_nvcomp)
 
     # Remove incorrect public dependency on the static cuda runtime We have to modify the
     # nvcomp-targets.cmake since these entries will cause a failure when rapids_cpm_find is called.
-    if(nvcomp_proprietary_binary)
+    if(nvcomp_proprietary_binary AND ${version} STREQUAL "2.3.2")
```

Comment:
We should use `VERSION_EQUAL` https://cmake.org/cmake/help/latest/command/if.html#version-equal